iscript/src/iscript/mac.py

```diff
@@ -494,6 +499,20 @@ def get_app_paths(config, task):
     return all_paths
 
 
+def filter_apps(all_paths, fmt, inverted=False):
+    """Filtres through a list of apps and returns a list of apps back based on filter.
+
+    Args:
+        all_paths: list of App objects
+        fmt: format name to filter
+        inverted (default: False): whether or not to invert the list.
+    """
```

Comment:
./src/iscript/mac.py:503:1: D413 Missing blank line after last section

flake8_docstrings just wants an empty line before the final `"""`.